tests/plugins/tableresize/manual/iebordercollapse.md

```diff
@@ -0,0 +1,26 @@
+@bender-ui: collapsed
+@bender-tags: bug, 2823, 4.14.0
+@bender-ckeditor-plugins: wysiwygarea, toolbar, table, tableresize, basicstyles, undo
+
+## Procedure
+1. Move mouse over the right table border (border after the last column).
+
+	### Expected
+
+	* Move cursor is visible.
+
+		Note: cursor can become visible slightly _after_ the right border.
```

Comment:
For me it is visible slightly before right border in IE11 :laughing: So maybe we should skip this note or mention that it can become visible slightly before/after the border.

Reply:
The more columns there are for each column the pillar is moved more to the left misaligning with column border. And when it comes to recalculating pillar position for IE, `pillarLeft -= pillarWidth;` solves this issue (so without substracting borders...). Tested on Win10 with IE11 `11.1098.17763.0` :thinking:

Reply:
> And when it comes to recalculating pillar position for IE, `pillarLeft -= pillarWidth;` solves this issue (so without substracting borders...)

That really doesn't make sense TBH, but yeah, it works…

Reply:
Yes, I was surprised as well. Not 100% sure why it works that way TBH. Some IE magic probably :thinking: